src/applications/lgy/coe/form/config/chapters/service/history.js

```diff
@@ -17,45 +14,21 @@ export const uiSchema = {
       itemName: 'Service Period',
       viewField: ServicePeriodView,
       keepInPageOnReview: true,
-      showSave: true,
```

Comment:
I remember there were a lot of UX discussions about the save button. I think they still want it to be seen. Please make sure before merging.

Reply:
I checked and it looks like it's being used in 2 places in the codebase. Our form includes another page with a list loop and it doesn't use `showSave` so I figured either they both should have it or neither of them should. Since it's only used in one form I removed it.